en/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md

```diff
@@ -8,14 +8,14 @@ tags: security
 lang: en
 ---
 
-We have released the uri gem version 0.10.0.1, 0.10.2, 0.11.1 and 0.12.1 that has a security fix for a ReDoS vulnerability.
+We have released the uri gem version 0.12.1, 0.11.1, 0.10.2 and 0.10.0.1 that has a security fix for a ReDoS vulnerability.
 This vulnerability has been assigned the CVE identifier [CVE-2023-28755](https://nvd.nist.gov/vuln/detail/CVE-2023-28755).
 
 ## Details
 
 A ReDoS issue was discovered in the URI component. The URI parser mishandles invalid URLs that have specific characters. It causes an increase in execution time for parsing strings to URI objects.
 
-The `uri` gem version 0.10.1, 0.10.2, 0.11.0, 0.12.0, and all versions 0.10.0 and prior are vulnerable for this vulnerability.
```

Comment:
The primary motivation behind this fix. I believe version 0.10.2 is not affected, as far as I understand.

Reply:
Thanks. `0.10.2` is not affected.